src/pint/observatory/__init__.py

```diff
@@ -469,30 +504,24 @@ def get_observatory(
     ----------
     name : str
         The name of the observatory
-    include_gps : bool or None, optional
-        Override UTC(GPS)->UTC clock correction.
-    include_bipm : bool or None, optional
-        Override TAI TT(BIPM) clock correction.
-    bipm_version : str, optional
-        Set the version of TT BIPM clock correction files.
+    apply_gps2utc : bool or None
+        Whether to apply BIPM Circular T to convert UTC(USNO)_GPS to UTC.
+        Default is None, which will use the the value set in observatories.json or True if not set there.
+        Specifying True or False here will override.
 
     .. note:: This function can and should be expanded if more clock
         file switches/options are added at a public API level.
 
     """
-    if include_bipm is not None or include_gps is not None:
-        site = deepcopy(Observatory.get(name))
-
-        if include_gps is not None:
-            site.include_gps = include_gps
-
-        if include_bipm is not None:
-            site.include_bipm = include_bipm
-            site.bipm_version = bipm_version
-
-        return site
+    if apply_gps2utc is None:
+        site = Observatory.get(name)
+    else:
+        # If we are overriding the gps2utc setting then  site and update apply_gps2utc
+        # Otherwise, will just use default setting for this site.
+        log.debug(f"Observatory.get(apply_gps2utc={apply_gps2utc})")
+        site = Observatory.get(name, apply_gps2utc=apply_gps2utc, overwrite=True)
```

Comment:
A warning here?

Reply:
I added one in `Observatory.get` so no need for one here.